views/support.ejs

```diff
@@ -9,6 +9,28 @@
     <link rel="stylesheet" type="text/css" href="/css/support.css" />
 </head>
 <body>
-    <h1>Contact Support</h1>
+    <div class = "bookshelf" >
+    <h1>How can we help you?</h1>
+
+    <form action="/login" method="POST" id="auth_form">
```

Comment:
form action should go to the support router instead, not login. Also need a post route in the support router.